### PR TITLE
feat(utils): added an internal endpoint and script for pd trigger

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#1040

### Description
- Added internal-patient endpoint, which triggers PD
- Added a script that allows to use this endpoint for patients en-masse

### Testing
- Local
  - [x] Trigger endpoint on Postman
  - [x] Run the script with a handful of patients

### Release Plan
- [ ] Merge this

### CODE REVIEW

File: packages/utils/src/bulk-trigger-patient-discovery.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const confirmationTime = dayjs.duration(10, "seconds");
+const delayTime = confirmationTime.asMilliseconds();
+const cxId = getEnvVarOrFail("CX_ID");
+const apiUrl = getEnvVarOrFail("API_URL");
+const CHUNK_DELAY_MAX_MS = dayjs.duration({ seconds: 1 }).asMilliseconds();
+const PATIENT_CHUNK_SIZE = 5;
+
+const patientIds: string[] = [];
+const rerunPdOnNewDemographics = true;
```
Comment (Orta21) [inline packages/utils/src/bulk-trigger-patient-discovery.ts:33]: Would probably default false

File: packages/utils/src/bulk-trigger-patient-discovery.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  };
+};
+
+async function main() {
+  try {
+    await displayInitialWarningAndConfirmation(patientIds.length);
+
+    const patientChunks = chunk(patientIds, PATIENT_CHUNK_SIZE);
+
+    for (const [j, patients] of patientChunks.entries()) {
```
Comment (Orta21) [inline packages/utils/src/bulk-trigger-patient-discovery.ts:47]: Why j instead of i?

**Review by Orta21 — APPROVED** [review]
Left a few comments but looks

File: packages/utils/src/bulk-trigger-patient-discovery.ts (hunk 2 above)
Comment (RamilGaripov) [reply packages/utils/src/bulk-trigger-patient-discovery.ts:47]: took this part from the inner loop of a different script and didnt change the letter haha

**Pull request merged**